docs/modules/integrate/pages/kafka-connector.adoc

```diff
@@ -103,6 +103,31 @@ properties.put("schema.registry.url", schemaRegistryUrl);
 Keep in mind that once the record deserialized, Jet still needs to know
 how to serialize/deserialize the record internally.
 
+== Embedded Avro Schema
+To use an embedded Avro schema for `GenericRecord`, you must provide the following as parameters for the Kafka producer and/or consumer:
```

Comment:
"embedded" might be misleading (embedded in records?). I prefer to name it "explicit"

Reply:
I had assumed that "embedded" was a recognised term; if it isn't (and if the equivalent recognised term isn't "explicit"), why not just Avro Schema?

Reply:
quick google search show that "embedded avro schema" means usually that schema is prepended to serialized data. This is not our case. Schema is not sent alongside the data, it is configured in properties.

Reply:
and to add even more confusion `GenericRecord` class also contains schema

Reply:
for Avro there are more that 2 possibilities:
* Avro Object Container File and similar approaches (prepends schema to data)
* Schema registry (prepends schema id to data)
* know schema upfront (nothing added to data) - OFC this is limited, but should be usable quite often when schemas are managed eg. in git repositories

Reply:
word "embedded" is deleted

Reply:
unqualified "Avro schema" is too generic. This section describes particular source of the schema (namely the properties). So I propose naming it "Explicit Avro Schema" or "Avro Schema in properties" or something like that. 

BTW, we also considered providing "explicit" avro schema via path to file on member (or maybe URL) so I prefer "explicit".

Reply:
ok. explicit avro schema